<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US">


<head>
<title>Columbia U01 LINCS Query</title>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<link href="css/lincs.css" rel="stylesheet">
<link href="style.css" rel="stylesheet">
</head>
<body>

	<div id="header-wrap">
		<div id="header">
			<h1>
				<a>NIH LINCS Program</a>
			</h1>
			<span class="tagline"></span>

			<div id="logo-nihlincs">
				<a> <span>Library of Integrated Network-Based Cellular
						Signatures</span>
				</a>
			</div>


		</div>
	</div>
	<!-- end header-wrap -->




	<div id="header1-wrap">

		<div id="header1">

           <div id="content" class="about">
			<h1 id="firstHeading" class="firstHeading">Columbia U01 LINCS Query</h1>


			<h2>
				<span class="mw-headline" id="Overview">Overview</span>
			</h2>
			<p>
				<a href="http://www.lincsproject.org/" class="external text"
					rel="nofollow" target="_blank">LINCS - Library of Integrated
					Network-based Cellular Signatures</a>
			</p>
			<p>The NIH LINCS Program supports a cellular network-based investigation into biology
				at a number of centers across the country.
				It catalogs changes to gene expression and other cellular processes resulting 
				from perturbation experiments. Various Centers employ sophisticated computational
				methods both prior to and after data collection to prioritize perturbations and
				analyse the results.</p>
			<p>This page provides for query and display of data
				generated by the Columbia LINCS Technology U01 and Computation U01
				Centers, both under the direction of Dr. Andrea Califano. 
				It provides experimental drug-drug synergy results, as well as
				computationally-derived similarity scores for drug mode of action.</p>
			<p>
				<b>Synergy (Experimental)</b> - Using any of a variety of assays, drugs
				are tested in combination to determine if they have a greater effect
				together than the sum of their separate effects. The synergy
				experiments are being carried out at the Columbia High-Throughput
				Screening Facility.
			</p>
			<p>
				<b>Similarity (Computational)</b> - The functional Mode of Action
				(fMoA) for each drug is a ranked list of transcription factor activities following drug perturbation.
				Calculation of the fMoA is described below.
				The similarity between the fMoA profiles of each
				drug can be calculated, e.g. using GSEA. The similarity scores are then
				used to predict whether the two compared drugs will be
				synergistic in their action. The hypothesis initially being tested
				is that drugs that are most dissimilar will have a greater
				probability of acting synergistically.
			</p>
			<p>
				<b>Functional Mode of Action (further detail)</b> - The Functional Mode of Action
				(fMoA) of drugs is calculated using the 
				<a href="http://wiki.c2b2.columbia.edu/califanolab/index.php/Software/VIPER" 
				class="external text" rel="nofollow" target="_blank">Viper</a> 
				program. Viper uses a cell-context (e.g. tissue) specific interaction network
				(interactome), reverse engineered from a large number of expression
				profiles using the 
				<a href="http://wiki.c2b2.columbia.edu/califanolab/index.php/Software/ARACNE" 
				class="external text" rel="nofollow" target="_blank">ARACNe</a> 
				algorithm, and also individual expression
				profiles generated for each tested drug. The interaction network
				contains the list of  regulated target genes (the <b>regulon</b>) for
				each transcription factor (TF) in the context of the tissue tested.
				Using the interaction network, Viper calculates the enrichment of
				the regulon of each TF in the list of differentially expressed genes
				produced for each drug. Thus the effect of each drug can be
				represented as the activity profile of its TFs, and this is termed
				the functional mode of action of the drug.
			</p>
			
			<h2>
				<span class="mw-headline" id="Currently_Available_Data">Currently
					Available Data</span>
			</h2>
			<p>
				This data is unpublished and released under terms of the <a
					href="http://wiki.c2b2.columbia.edu/workbench/images/f/f5/Columbia_LINCS_Data_Sharing_and_Citation_Guidelines_2014-10-04.pdf"
					 
					title="Columbia LINCS Data Sharing and Citation Guidelines 2014-10-04.pdf">
					draft LINCS data sharing and citation agreement</a>, which is included
				with the LINCS Query component.
			</p>
			<ul>
				<li>Synergy - highest synergy value obtained for each pairwise
					titration
					<ul>
						<li>LNCaP - 100x10</li>
						<li>MCF7 - 14x10 (preliminary)</li>
					</ul>
				</li>
				<li>Similarity
					<ul>
						<li>MCF7 - full results.</li>
					</ul>
				</li>
			</ul>
			 
			<h2>
				<span class="mw-headline" id="LINCS_Query_and_Display">The Columbia LINCS
					Query and Display Interface</span>
			</h2>
			<p>Query for experimental synergy results for tested drug pairs,
				or computational drug similarity results based on gene expression
				experiments. For synergy experiments, drug 1 and drug 2
				are drawn from separate lists of compounds.  Drug 1
				corresponds to a list of 100 "sensitizer" drugs chosen to maximally
				orthogonal to each other in their mode of action. Drug 2 corresponds
				to a set of 10 "effector" compounds of known mode of action.</p>
			 
			<h2>
				<span class="mw-headline"
					id="Experimental_and_Computational:_Search_Controls_in_Common">Experimental
					and Computational: Search Controls in Common</span>
			</h2>
			<ul>
				<li><b>Tissue Type</b> - The tissue from which the tested cell
					line was originally derived. More than one tissue can be chosen for
					a single query.</li>
				<li><b>Cell Line</b> - The cell line tested in a particular
					experiment. More than one cell line can be chosen for a single
					query (multiple selection), subject to any constraint on "Tissue
					Type".</li>
				<li><b>Drug 1</b> - Entries stored as "drug 1" in a pair,
					subject to previous choices of tissue type or cell line, if any. <b>One
						or more drugs (multiple selection) must be chosen</b>. </li>
				<li><b>Drug 2</b> - Entries stored as "drug 2" in a pair, given
					current constraints on tissue type, cell line, and drug 1. More
					than one drug can be chosen for a single query (multiple
					selection). </li>
			</ul>
			<ul>
				<li><b>Dynamic Search on Drug 1</b> - currently the number of
					drugs in "Drug 1" can be considerably larger than in "Drug 2". For
					this reason, a dynamic search box is offered above the "Drug 1"
					selection. As each character is typed, the "Drug 1" list will be
					updated to only show matching drugs.
					<ul>
						<li>The match is not case-sensitive.</li>
						<li>The match can occur anywhere in the drug name.</li>
					</ul></li>
			</ul>
			<p>
				<br />
			</p>
			<h2>
				<span class="mw-headline"
					id="Experimental_vs_Computational:_Mode-Specific_Search_Controls">Experimental
					vs Computational: Mode-Specific Search Controls</span>
			</h2>
			<p>Radio buttons are used to select whether to query for
				experimental or computational results. </p>
			<p>The controls specific to each type are:</p>
			<h3>
				<span class="mw-headline" id="Experimental">Experimental</span>
			</h3>
			<ul>
				<li><b>Assay Type</b> (phenotypic readout): May include viability,
					proliferation, apoptosis, cell morphology, oxidization state, etc.</li>
				<li><b>Synergy Measure</b> May include Excess-over-Bliss, Excess over
					highest single agent, Combination Index, etc.</li>
			 
			</ul>
			<h3>
				<span class="mw-headline" id="Computational">Computational</span>
			</h3>
			<ul>
				<li><b>Similarity Algorithm</b>  May inlcude MARINa (GSEA-2), IDEA, DEMAND,
					combined, etc. This entry will combine the algorithm used with
					the similarity method used, e.g. GSEA-2, in the case where an
					algorithm has more than one possible measure.</li>
			</ul>
		 
			<h2>
				<span class="mw-headline" id="Query_Controls">Query Controls</span>
			</h2>
			<ul>
				<li><b>Max Results</b> (checkbox and adjacent text field) -
					when checked, use the number <i>n</i> entered in the adjacent text
					field to restrict the returned results to those with the top <i>n</i>
					scores.</li>
				<li><b>Search</b> - Search the database for all results
					matching the constraints, and display the result in the spreadsheet
					view.</li>
				<li><b>Reset</b> - Reset all constraints to their default
					values and clear the lower spreadsheet display.</li>
				<li><b>Color Gradient for Score</b> (checkbox) - if checked,
					color the cells in the Score column using a heat-map type
					scheme.</li>
			</ul>

             </div>  <!-- end about -->

		</div>
		<!-- end pagebody -->

	</div>
	<!-- end pagebody-wrap -->


	<div id="footer-wrap">
		<div id="footer">&copy;2014, Columbia University. All rights
			reserved.</div>
	</div>
	<!-- end footer-wrap -->






</body>
</html>
